businessRules/ideas.js

```diff
@@ -1,9 +1,65 @@
-const { getAllIdeas } = require('../models/ideas');
+const {
+    getAllIdeas,
+    createIdea,
+    updateIdea,
+    deleteIdea,
+} = require('../models/ideas');
 
 function getIdeas() {
     return getAllIdeas();
 }
 
+function sortedByDonationSumIdeas() {
+    const ideasList = getIdeas();
+
+    const sortedIdeasList = ideasList.sort(
+        (a, b) => a.totalDonationSum - b.totalDonationSum
+    );
+    return sortedIdeasList;
+}
+
+function createIdeas(header, description, askedSum, userId) {
+    if (header === null) {
```

Comment:
Instead of a strict comparison against `null`, you can simply use `!`, e.g.:
```js
if (!header) {...}
```

Reply:
I've used similar way of testing if true below in that function, why I've never thought about using this way of testing if true in the beggining?
Anyway fixed and tested.